.github/workflows/dormant/windows.yml

```diff
@@ -0,0 +1,90 @@
+name: Windows builds
```

Comment:
Suggest postfixing this new dormant file with msys2: `./dormant/windows-msys2.yml` 

(This will at least help me; sometime I mass-move files in and out of `dormant` when I'm testing changes...)

Reply:
Done :)